ditto/writers/opendss/write.py

```diff
@@ -3801,6 +3802,7 @@ def write_master_file(self, model):
                         fp.write(
                             " basekV={volt}".format(volt=obj.nominal_voltage * 10 ** -3)
                         )  # DiTTo in volts
+                        self._baseKV_.add(obj.nominal_voltage * 10 ** -3)
```

Comment:
isn't the setting of the _baseKV_ done earlier (e.g. line 636 in the master branch)?

Reply:
`self._baseKV_` is a set of nominal voltages used to write out the `Set Voltagebases=[...]` command for openDSS. This line adds the base kV value to the set for the  `New Circuit` / `PowerSource`